src/SFML/Window/Context.cpp

```diff
@@ -64,6 +64,11 @@ Context::~Context()
     }
 }
 
+////////////////////////////////////////////////////////////
+Context::Context(Context&& other) noexcept = default;
+
+////////////////////////////////////////////////////////////
+Context& Context::operator=(Context&& other) noexcept = default;
```

Comment:
Please put two newlines between these. Look at the rest of this .cpp and try to copy the code style you see in there. Also remove the `other` parameter. Because it's not used let's not even name it. Remove it from the .hpp file too please.